src/components/Sum/Sum.jsx

```diff
@@ -1 +1,5 @@
-// export const Sum = () => ();
+export const Sum = ({ a = 0, b = 0 }) => (
+  <p>
+    {`Sum of ${a} and ${b} is ${a + b}`}
```

Comment:
While this works, remember that you do not have to open (and close) a string while in JSX, so the code below works exactly the same way but is easier to read:
```javascript
<p>
  Sum of {a} and {b} is {a + b}
</p>
```

Reply:
It doesn't work with lint, so I leave it as before.